prober/http.go

```diff
@@ -123,30 +141,76 @@ func (h *HTTP) calculateInterval() time.Duration {
 }
 
 func (h *HTTP) sendRequest(ctx context.Context) HTTPResult {
-	req, _ := http.NewRequestWithContext(ctx, http.MethodGet, h.URL, nil)
-	if h.Host != "" {
-		req.Host = h.Host
+	var startTime time.Time
+	var finishTime time.Time
+	var dnsStartTime time.Time
+	var dnsDoneTime time.Time
+	var connectStartTime time.Time
+	var connectDoneTime time.Time
+	var tlsHandshakeStartTime time.Time
+	var tlsHandshakeDoneTime time.Time
+	httpTrace := &httptrace.ClientTrace{
```

Comment:
please extract the tracing codes into another method.

Reply:
will be refactored in another PR

Reply:
@m-yosefpor do you have any ETA on this?